package.json

```diff
@@ -44,18 +44,18 @@
     "typescript": "^4.1.5"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.7.0",
-    "@typescript-eslint/parser": "^5.7.0",
-    "eslint": "^7.25.0",
-    "eslint-config-prettier": "^4.1.0",
-    "eslint-plugin-import": "^2.25.4",
+    "@typescript-eslint/eslint-plugin": "^5.23.0",
+    "@typescript-eslint/parser": "^5.23.0",
+    "eslint": "^8.15.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-lodash": "^7.1.0",
-    "eslint-plugin-prettier": "^3.0.1",
-    "eslint-plugin-react": "^7.28.0",
-    "eslint-plugin-react-native": "^3.7.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react-native": "^4.0.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
-    "prettier": "^2.5.1",
-    "typescript": "^4.1.5"
+    "prettier": "^2.6.2",
+    "typescript": "^4.6.4"
```

Comment:
Wanna drop this one down to ^4.1.0? Since we're pinned back for the mongoose-tsgen issue?

Reply:
Done!